elements/pl-file-upload/pl-file-upload.js

```diff
@@ -34,16 +35,22 @@ window.PLFileUpload.prototype.initializeTemplate = function() {
         url: '/none',
         autoProcessQueue: false,
         accept: function(file, done) {
-            if (_.includes(that.acceptedFiles, file.name)) {
+            // fuzzy case:
+            const fileNameLowerCase = file.name.toLowerCase();
+            if (_.includes(that.acceptedFilesLowerCase, fileNameLowerCase)) {
                 return done();
             }
             return done('invalid file');
         },
         addedfile: function(file) {
-            if (!_.includes(that.acceptedFiles, file.name)) {
+            // fuzzy case match
+            const fileNameLowerCase = file.name.toLowerCase();
+            if (!_.includes(that.acceptedFilesLowerCase, fileNameLowerCase)) {
                 that.addWarningMessage('<strong>' + file.name + '</strong>' + ' did not match any accepted file for this question.');
                 return;
             }
+            const acceptedFilesIdx = that.acceptedFilesLowerCase.indexOf(fileNameLowerCase);
+            const acceptedName = that.acceptedFiles[acceptedFilesIdx];
```

Comment:
```suggestion
            const acceptedName = that.acceptedFiles[acceptedFilesIdx];
            if (file.name !== acceptedName) {
                that.addWarningMessage(`<strong>${file.name}</strong> was automatically renamed to <strong>${acceptedName}</strong>.`);
            }
```
Ah right, so it would be something like this if we wanted to give a renaming message. Not sure if we want to do this, just a thought.

Reply:
I'm inclined to not show any warnings. Students using Windows or MacOS are probably unaware of the case of their filename extensions, for example, and so they are going to get these warnings that won't make any sense to them.

Reply:
I guess that's reasonable. If a warning implies that the user should take action, this may be unnecessarily confusing now.